IFC 4.3/ProjectSetup_1/UT_ProjectSetup_1.ifc

```diff
@@ -0,0 +1,48 @@
+ISO-10303-21;
+
+/* ------- This file was produced manually for IFC Infra Extension Deployment project. --------- */
+/* ------- The file should be come with a README.md with supplementing info. ------------------- */
+/* ------- Stefan Jaud, stefan.jaud@tum.de, 2020.07.15. ---------------------------------------- */
+
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [NotDefinedYet]'), '2;1');
+FILE_NAME ('UT_ProjectSetup_1.ifc', '2020-07-15T18:00:00', (''), (''), 'Stefan Jaud manually', 'SublimeText 3.0', '');
+FILE_SCHEMA (('IFC4X3_RC1'));
+ENDSEC;
+
+DATA;
+
+/* ------- General definitions ----------------------------------------------------------------- */
+#1 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
```

Comment:
Space is fine.

> 5.6 Token separators
> 
> A token separator is an element that separates two tokens. Token separators are space, the explicit print control directives, and comments. A token separator may appear between the terminals or non-terminals of the productions of Table 3. Any number of token separators may appear wherever one token separator may appear. A token separator shall not appear within tokens except that explicit print control directives may also appear within binaries and within strings. Print control directives are defined in clause 13.
> NOTE      Space is the only whitespace character that separates tokens. Line-delimiters such as line feed or carriage return and other control characters such as form feed or character tabulation (tab) may appear in the exchange structure but are required by 5.2 to be ignored when processing the exchange structure. Consequently, line breaks may appear anywhere within the structure, including within tokens.

Reply:
I am a bit skeptical. PARAMETER_LIST itself is a token and "A token separator shall not appear within tokens".

Reply:
Hm I agree the wording is not very clear. I would expect "A token separator may appear between the terminals or non-terminals" to take priority and expect "A token separator shall not appear within tokens" to imply "... within terminal tokens". After all the entire file is also an `EXCHANGE_FILE` token, meaning that token separators may not exist at all if you take "A token separator shall not appear within tokens" literally.

Reply:
Yes, that sounds reasonable. So, just mentally add "of table 2" to "A token separator shall not appear within tokens" resulting in "A token separator shall not appear within tokens of table 2". This would do the trick and to be honest it is kind of implied that this is the case in "A token separator may appear between the terminals or non-terminals of the productions of Table 3. "

Reply:
I suppose that the spaces are fine? No change needed?